package.json

```diff
@@ -32,8 +32,8 @@
     "dist"
   ],
   "engines": {
-    "node": ">=20.0.0",
-    "pnpm": "8"
+    "node": ">=22.11.0",
```

Comment:
Given this change, can you also update https://github.com/wundergraph/cosmo/blob/af0ae3ba2d0afb4a637d101e93e06644cbafb3c4/CONTRIBUTING.md?plain=1#L13-L14 to reflect the new versions?

Reply:
Good catch